src/Reflection/Php/PhpFunctionReflection.php

```diff
@@ -256,4 +256,13 @@ public function getAsserts(): Assertions
 		return $this->asserts;
 	}
 
+	public function getDocComment(): ?string
+	{
+		$docComment = $this->reflection->getDocComment();
```

Comment:
This isn't always correct, look where functionReflectionFactory is called. It can either be the method's PHPDoc, or the one in stub files.